src/goto-instrument/contracts/dynamic-frames/dfcc_lift_memory_predicates.cpp

```diff
@@ -0,0 +1,425 @@
+/*******************************************************************\
+
+Module: Dynamic frame condition checking for function contracts
+
+Author: Remi Delmas, delmasrd@amazon.com
+Date: August 2022
+
+\*******************************************************************/
+
+#include "dfcc_lift_memory_predicates.h"
+
+#include <util/cprover_prefix.h>
+#include <util/format_expr.h>
+#include <util/graph.h>
+#include <util/pointer_expr.h>
+#include <util/replace_symbol.h>
+#include <util/symbol.h>
+
+#include <goto-programs/goto_model.h>
+
```

Comment:
Is this order right?

Reply:
yes, that's the order produced by the clang-format rules.